ui/main/src/app/views/customCardList/resultTable/ResultTableExport.ts

```
@@ -0,0 +1,112 @@
+                            break;
+                        default:
+                            if (cellValue?.text) cellValue = cellValue.text;
+                            row[column.headerName] = cellValue;
+                    }
+                }
+            });
+            return row;
+        });
+    }
+
+    private addResponseFieldsToRow(responseFieldName: string, responses: {color: string; name: string}[], row: {}) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)